pages/common/assets/sass/style.scss

```diff
@@ -28,6 +28,20 @@
   margin-bottom: $govuk-gutter;
 }
 
+table {
+  .pull-right {
+    width: 15%;
+  }
+}
+
+p.subtitle {
+  font-size: 1.1em;
+}
+
+.margin-bottom {
```

Comment:
this feels a bit icky

Reply:
I don't disagree, but also I'm not inclined to go and find every instance of a details element to work out how doing it differently would affect things.